src/v1/components/inputs/SearchBox/MiniSearchBox.tsx

```diff
@@ -0,0 +1,260 @@
+  /**
+   * The value of the `input` element, required for a controlled component.
+   */
+  value?: unknown;
+}
+
+export interface MiniSearchBoxProps extends SearchInputBaseProps {
+  /**
+   * If true, a loading indicator will be visible
+   */
+  loading?: boolean;
+  /**
+   * If true, a arrow down button will be visible
+   */
+  showDownArrowButton?: boolean;
+  /**
+   * Callback fired when the search button is clicked
+   *
+   * `(event: React.MouseEvent<HTMLButtonElement>) => void;`
+   */
+  onSearch?: React.MouseEventHandler<HTMLButtonElement>;
+  /**
+   * Callback fired when the down arrow button is clicked
+   *
+   * `(event: React.MouseEvent<HTMLButtonElement>) => void;`
+   */
+  onTogglePopOver?: React.MouseEventHandler<HTMLButtonElement>;
+}
+
+const MiniSearchBox = (props: MiniSearchBoxProps) => {
+  const {
+    loading = false,
+    showDownArrowButton = false,
+    onSearch,
+    onTogglePopOver,
+    onFocus,
+    onBlur,
+    ...inputProps
+  } = props;
+
+  const [isFocused, setIsFocused] = useState(false);
+
+  const handleOnFocus = (
+    event: React.FocusEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    setIsFocused(true);
+
+    if (onFocus) {
+      onFocus(event);
+    }
+  };
+
+  const handleOnBlur = (
+    event: React.FocusEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    setIsFocused(false);
+
+    if (onBlur) {
+      onBlur(event);
+    }
+  };
+  // const internalTogglePopOverButtonRef = useRef<HTMLButtonElement>(null);
+
+  // useImperativeHandle(props.togglePopOverButtonRef, () => ({
+  //   togglePopOver: () => {
+  //     const syntheticMouseEvent = new MouseEvent("click", {
+  //       bubbles: true,
+  //       cancelable: true,
+  //       view: window,
+  //     });
+
+  //     internalToggleButtonRef.current?.dispatchEvent(syntheticMouseEvent);
+  //   },
+  // }));
+
+  // return (
+  //   <FlexBox
+  //     direction="row"
+  //     sx={{
+  //       width: 390,
+  //       borderStyle: "solid",
+  //       borderWidth: 1,
+  //       borderColor: "#989898",
+  //       borderRadius: 1,
+  //       paddingInline: 1.5,
+  //       justifyContent: "center",
+  //     }}
+  //   >
+  //     <MUIInputBase
+  //       type="search"
+  //       sx={{ width: "100%" }}
+  //       slotProps={{
+  //         input: {
+  //           sx: { padding: 0, paddingRight: 1, height: "20px" },
+  //         },
+  //       }}
+  //       // {...inputProps}
+  //     />
+  // <FlexBox direction="row" spacing={0.5}>
+  //   <IconButton size="small" onClick={onSearch} color="primary">
+  //     {loading ? (
+  //       <MUICircularProgress />
+  //     ) : (
+  //       <SearchIcon fontSize="inherit" />
+  //     )}
+  //   </IconButton>
+  //   <IconButton
+  //     // ref={internalToggleButtonRef}
+  //     size="small"
+  //     onClick={onTogglePopOver}
+  //   >
+  //     <DownArrowIcon fontSize="inherit" />
+  //   </IconButton>
+  // </FlexBox>
+  //   </FlexBox>
+  // );
```

Comment:
Is this because of the ref bug you mentioned? If so could you add a comment with a link to the bug description?